sql/src/test/java/io/crate/operation/udf/JavascriptUserDefinedFunctionTest.java

```diff
@@ -192,4 +194,35 @@ public void testAccessJavaClasses() throws Exception {
             "function f(x) { var File = Java.type(\"java.io.File\"); return x; }");
         assertEvaluate("f(x)", 1L, Literal.of(1L));
     }
+
+    @Test
+    public void testEvaluateOnStringInputs() throws Exception {
```

Comment:
please clarify somehow that these tests are testing the bytesRef/string conversions